src/pretix/helpers/compat.py

```diff
@@ -1,9 +1,48 @@
 import datetime
 import sys
+from django.forms import Form
```

Comment:
Please separate the group of "3rd party imports" and the standard library with a blank line (PEP8).